config/kernel/linux-media-edge.config

```diff
@@ -10515,6 +10517,7 @@ CONFIG_DEBUG_MISC=y
 #
 # Compile-time checks and compiler options
 #
+CONFIG_AS_HAS_NON_CONST_LEB128=y
```

Comment:
I assume these are automatic adjustments? Just asking because this is balbes playground.

Reply:
Auto, yes.